src/System.Net.Http/tests/FunctionalTests/CancellationTest.cs

```diff
@@ -28,7 +28,7 @@ public CancellationTest(ITestOutputHelper output)
         [InlineData(false, true)]
         [InlineData(true, false)]
         [InlineData(true, true)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19038")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.NetFramework, "uap: dotnet/corefx #20010, netfx: dotnet/corefx #19038")]
```

Comment:
Yes I added this new capabilities to ActiveIssue like 2 days ago. So it can be used to disable a test for a specific target framework moniker, as Dan pointed above.